Add a command to install the executable directly from the source repository

ppetr opened the pull request:
Also some minor fixes/improvements of links.

hansmi reviewed (changes requested):
Got a few comments. Please let me know if I should take care of them.

ppetr commented on wp2reg-luxws-exporter.service:
```diff
@@ -0,0 +1,27 @@
+ExecStart=/usr/local/sbin/luxws-exporter --controller.address=192.168.0.1:8214 --controller.address.http=192.168.0.1:80 --controller.language=cz --web.listen-address=localhost:9101
+Restart=always
+RestartSec=30
+; The fields below are optional, but strongly recommended: The service is put
```
Mostly I agree on this and I'll update the file as you suggest. The only case where I'd argue is the `DynamicUser`: This significantly increases security of the service, and at the same time it's something that isn't a purely policy decision, because it might need other settings depending on which file a service accesses etc. So at least I'd keep this as a comment that can be uncommented by users.

hansmi replied:
I'd argue that compiling and installing binaries as root (and likely talking to remote services for fetching modules) is considerably worse than anything `DynamicUser` could ever provide. ;-)

On a serious note I'm fine with leaving `DynamicUser` in there. If you want I can also follow-up to add the systemd unit to the pre-built packages.

ppetr replied:
Point taken :sweat_smile:

ppetr commented on wp2reg-luxws-exporter.service:
ACK

ppetr replied:
Done.

ppetr commented on README.md:
```diff
@@ -25,16 +26,24 @@ directory for details.
+It's also possible to produce custom builds directly using [Go][golang] or
+[GoReleaser][goreleaser].
+
+With [Go 1.16][go1.16] or later the following command installs `luxws-exporter`
```
ACK

ppetr replied:
Done.

ppetr commented on wp2reg-luxws-exporter.service:
```diff
@@ -0,0 +1,27 @@
+[Service]
+Type=simple
+; ===> Set the path and flags according to your environment. <================
+ExecStart=/usr/local/sbin/luxws-exporter --controller.address=192.168.0.1:8214 --controller.address.http=192.168.0.1:80 --controller.language=cz --web.listen-address=localhost:9101
```
Done. I was wondering if it'd make sense to split it into multiple environment variables, but for the time being I kept it the same way as your example.

ppetr commented:
I addressed your comments. I created a new commit for them, since I'm using mercurial-git and it breaks the repo when attempting to rewrite history, so feel free to squash them into a single one.

hansmi reviewed (approved):
Looks good to me. I'll squash the commits locally.
